recipe/meta.yaml

```diff
@@ -13,26 +13,58 @@ source:
     - 0001-Don-t-hardcode-the-CoreFoundation-path.patch
 
 build:
-  number: 1
+  number: 2
   run_exports:
     - {{ pin_subpackage("aws-c-common", max_pin="x.x.x") }}
 
-requirements:
-  build:
-    - cmake
-    - {{ compiler('c') }}
-    - ninja
+outputs:
+  - name: aws-c-common
+    script: build-shared.sh  # [unix]
+    script: bld-shared.bat  # [win]
+    build:
+      activate_in_script: True
+      run_exports:
+        - {{ pin_subpackage("aws-c-common", max_pin="x.x.x") }}
+    requirements:
+      build:
+        - cmake
+        - {{ compiler('c') }}
+        - ninja
+    test:
+      commands:
+        - test -f $PREFIX/lib/libaws-c-common${SHLIB_EXT}  # [unix]
+        - test ! -f $PREFIX/lib/libaws-c-common.a  # [unix]
+        - test -f $PREFIX/include/aws/common/config.h  # [unix]
+        - test -f $PREFIX/lib/cmake/AwsCFlags.cmake  # [unix]
+        # Check for the non-existence of -Werror in the CFLAGS
+        - "! grep Werror $PREFIX/lib/cmake/AwsCFlags.cmake"  # [unix]
+        - if not exist %PREFIX%\\Library\\bin\\aws-c-common.dll exit 1  # [win]
+        - if exist %PREFIX%\\Library\\lib\\aws-c-common.lib exit 1  # [win]
```

Comment:
This should exist. You need a thin static library on Windows even for dynamic linkage.

Reply:
So, since the tests pass, it means we have a `.dll` but not a `.lib`. Not sure why?

Reply:
Because of a weird `conda-build` behaviour not running part of the build.